main.py

```diff
@@ -1,58 +1,67 @@
 import requests
 import jmespath
 from bs4 import BeautifulSoup
-import re
 
-final_results = []
-titles = []
-posters = []
-rates = []
-genres = []
-directors = []
-results = {}
+HTTP_SUCCES = 200
+MOVIES_URL = 'https://www.filmweb.pl/serials/search?endRate=10&orderBy=popularity&descending=true&startRate=9{}'
 
-response = requests.get('https://www.filmweb.pl/serials/search?endRate=10&orderBy=popularity&descending=true&startRate=9')
-soup = BeautifulSoup(response.content, 'html.parser')
-results_list = soup.find(class_="resultsList hits")
+try:
+    n = int(input("How many pages do you want to scrape?"))
+except ValueError:
+    raise Exception("Invalid value")
 
-results_list_posters = results_list.find_all(class_='poster__image')
-for poster in results_list_posters:
-    posters.append(jmespath.search('content', poster))
 
-results_list_titles = results_list.find_all(class_='filmPreview__title')
-for title in results_list_titles:
-    titles.append(title.get_text())
+def find_movies_of_n_pages(n=n):
 
-results_list_rates = results_list.find_all(class_="rateBox__rate")
-for rate in results_list_rates:
-    rates.append(rate.get_text())
+    def find_movies(page):
+        response = requests.get(MOVIES_URL.format(f'&page={page}'))
+        if response.status_code == HTTP_SUCCES:
```

Comment:
Requests library defines all status codes for you, so there is no need to create your own constant. `requests.codes.OK` is for code 200.